sdk/ml/azure-ai-ml/swagger/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2023-02-01-preview/registries.json

```diff
@@ -0,0 +1,874 @@
+        },
+        "sku": {
+          "description": "Sku details required for ARM contract for Autoscaling.",
+          "$ref": "#/definitions/PartialSku"
+        },
+        "tags": {
+          "description": "Resource tags.",
+          "type": "object",
+          "additionalProperties": {
+            "type": "string",
+            "x-nullable": true
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "PartialManagedServiceIdentity": {
+      "description": "Managed service identity (system assigned and/or user assigned identities)",
+      "type": "object",
+      "properties": {
+        "type": {
+          "description": "Managed service identity (system assigned and/or user assigned identities)",
+          "$ref": "../../../../../common-types/resource-management/v3/managedidentity.json#/definitions/ManagedServiceIdentityType"
+        },
+        "userAssignedIdentities": {
+          "description": "The set of user assigned identities associated with the resource. The userAssignedIdentities dictionary keys will be ARM resource ids in the form: '/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.ManagedIdentity/userAssignedIdentities/{identityName}. The dictionary values can be empty objects ({}) in requests.",
+          "type": "object",
+          "additionalProperties": {
+            "$ref": "#/definitions/PartialUserAssignedIdentity"
+          }
+        }
+      },
+      "additionalProperties": false
+    },
+    "PartialUserAssignedIdentity": {
+      "type": "object",
+      "additionalProperties": false
+    },
+    "PartialSku": {
+      "description": "Common SKU definition.",
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "format": "int32",
+          "description": "If the SKU supports scale out/in then the capacity integer should be included. If scale out/in is not possible for the resource this may be omitted.",
+          "type": "integer"
+        },
+        "family": {
+          "description": "If the service has different generations of hardware, for the same SKU, then that can be captured here.",
+          "type": "string"
+        },
+        "name": {
+          "description": "The name of the SKU. Ex - P3. It is typically a letter+number code.",
+          "type": "string"
+        },
+        "size": {
+          "description": "The SKU size. When the name field is the combination of tier and some other value, this would be the standalone code.",
+          "type": "string"
+        },
+        "tier": {
+          "description": "This field is required to be implemented by the Resource Provider if the service has more than one tier, but is not required on a PUT.",
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/SkuTier"
+        }
+      },
+      "additionalProperties": false
+    },
+    "Registry": {
```

Comment:
This class needs a new property called managedResourceGroupTags

Diff: https://github.com/Azure/azure-sdk-for-python/pull/26977/files#diff-3a56962ce5e7fd7bd3fdbbfde0df665e90d34b8c5468f2aadcb21df98a68f9fb
Issue: https://dev.azure.com/msdata/Vienna/_workitems/edit/2176856

Reply:
@MilesHolland Based on our offline discussion, added managedResourceGroupTags to registries.json